pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
> How fast can we get an origin image with the plugin?

all depends what we're willing to live with.  i can build and publish what we have today if we want (though I do need to make sure introducing the pipeline and sync plugins to our image don't break any of our existing jenkins flows)